doc/tokens/tokentypes/daypassword.rst

```diff
@@ -0,0 +1,43 @@
+.. _daypassword_token:
+
+Day Password Token
+-------------------
+
+The day password token is a modification of the :ref:`totp_token`. The difference between the
+:ref:`totp_token` and day password token is, that the PIN of the day password token can be used more
+then ones. For example you can use the day password token as a day password, in this way you get every
+day a new password for the day.
+Because the day password token basing on :ref:`totp_token` it works with the same Algorithms. You
+can find the Algorithm hir `RFC6238 <https://tools.ietf.org/html/rfc6238>`_.
+
+The day password token algorithm has some parameter, like if the generated OTP value will
+be 6 digits or 8 digits or if the SHA1 or the SHA256 hashing algorithm is
+used. You can freely set the timestep. For example you can set the timestep 3600, this way you get
+a pin for one hour.
+
+The day password token implements the :ref:`authenticate mode <authentication_mode_authenticate>`.
+With a suitable :ref:`policy_challenge_response` policy, it may also be used
+in the :ref:`challenge mode <authentication_mode_challenge>`.
+
+
+Software tokens
+~~~~~~~~~~~~~~~
+
+Experiences
+...........
+
+The token will create a QR code for enrollment. You can scene this code with an authenticator
+app on your smartphone.
+
+The DayPassword token can only be enrolled with an authenticator app that supports the new token.
+
+.. Note :: The link in the QR code lucks lick this:
+ otpauth://daypassword/DayPassword0000536F?secret=NW6IT27XNRXOAZEP4GILXB3GCKCANVQR&issuer=privacyIDEA&algorithm=SHA256&digits=6&period=86400
+
+Enrollment
+~~~~~~~~~~
+
+Default settings for day password tokens can be configured at :ref:`daypassword_token_config`.
```

Comment:
I would focus here on the enrollment policies, not the system-wide token config (we will remove this in the future and focus on policies only).

Reply:
https://github.com/privacyidea/privacyidea/pull/3602/commits/1d33e4a5414b7c3962159450ba40a6e9f9e29488